### ISSUE TITLE
Double click to select project

### ISSUE THREAD

**Issue by Res260** [caido/caido#49]
### Community Note

- Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help us know what to work on
- Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request

### Description

So I just launched caido for the first time 5 minutes ago and I know these are nitpicky and obviously not a priority, but I feel they should be noted nonetheless :) 

The selected project on the top right feels like a button, but isn't. I would have expected (and appreciated) for it to redirect to the projects page when clicked.
![image](https://user-images.githubusercontent.com/14599855/140449637-204ab3fa-2ffd-4d0f-b847-f72621a2b7be.png)

In the projects page, I would also have expected that double-clicking on a project on the top to automatically select it.
![image](https://user-images.githubusercontent.com/14599855/140449815-76d33f23-33fd-4fc2-a303-c7fd72864d00.png)

In the "convert" page, the contrast ratio for the encoding labels have a very low contrast, and I'm having trouble reading some of them when my night shift is on (I do not have any vision problem). [This small script](https://github.com/davidjohnstone/contrast-checker) confirms that the contrast is too low:
![image](https://user-images.githubusercontent.com/14599855/140451061-3727e079-3813-4ad7-bb1e-0bf89de2a8b3.png)

In the "replay" page, when specifying the host and port, I'd expect the "Use TLS" to be checked by default, because the default value is port 443. I'd also expect the "Use TLS" button to be automatically re-selected when entering port 443, and port 443 be entered when checking "Use TLS", but still having the possibility to uncheck or change the port number, if some evil soul decides to listen for HTTP traffic on port 443.
![image](https://user-images.githubusercontent.com/14599855/140452658-59ffd384-d690-42c5-a840-f921c75a2ab7.png)

When trying the "replay" feature, I actually forgot to set up the CA on my windows machine. When I pressed "send", the UI didnt show any error (the console did though) and the arrows to see the different requests were activated, like the request was done correctly:
![image](https://user-images.githubusercontent.com/14599855/140452650-54490312-e7d2-4424-bf5d-d88c4eb3c638.png)

**Comment by Rhynorater** [comment]
I'm gonna add a particular :+1: to the things about the project UI and the replay port 443 UI. Those were things I experienced as well.

**Comment by Corb3nik** [comment]
Great issues :) 

Regarding your last comment about using the replay feature without installing the CA cert: I believe that's a false positive, since you don't need to install the CA cert to use the replay feature.

I think you sent a request to perdu.com:443 without any body, so its normal that you'd receive an empty response. (We could probably make that clearer, e.g. showing a "No response received" in the `Response` section maybe? Other ideas are welcome)

As for the logs TLS error logs that you're seeing, could they have been passive requests coming from your browser while you were testing the replay? The fact that those errors popped up while you were testing the replay is probably a coincidence.

**Comment by Res260** [comment]
Yeah, when playing around I realised what that feature was and felt a little dumb not realising that 😂 A "no respons received" sounds like a good idea.

For the TLS errors, yes, I double checked and I accidentally routed all requests to caido instead of only perdu.com, oops 😂

**Comment by Corb3nik** [comment]
Renaming this issue to `Double click to select project` since most of these issues here don't apply anymore :)

**Comment by Sytten** [comment]
I would also put the "recent" projects fast select on the top right maybe?
